### ISSUE TITLE
Find Count keyword shows 1 less than what log should be

### ISSUE THREAD

**Issue by cachly** [cachly/bug-tracker#1049]
**[Original report](https://bitbucket.org/zedsaidstudio/cachly/issues/1049) by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

It was after I submitted them.  When I originally drafted the log and did the preview, the numbers were correct.  Then I saved them offline.  Later I went back, finished writing my logs and submitted them.  That is when it picked up the last number and applied it to all the logs.  It also counted DNF and NM logs as a find number even though I did not have that quick text entered into those logs.
